### PR TITLE
#3509 Minor chart style changes

### LINKED ISSUE #3509: Minor chart updates
## Describe the bug

On my screen the bar chart and line chart styles look a little funny:

image

image

### To reproduce

1. Go to fridge detail page

### Expected behaviour

space between bars, line chart filling the space better

### Proposed Solution

TBD
### Version and device info

- App version: 7.0 rc 3
- Tablet model:
- OS version:

### Additional context
n/a

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3509 

## Change summary

- Slight chart style updates
- Also ended up playing with the tick labels
- Also ended up playing a bit with the sensor status
- Will probably now look funny on someone elses machine 🤷 

## Testing

- [ ] Looks good?

### Related areas to think about

image

image

### CODE REVIEW

**Comment by joshxg** [comment]
Hopefully these changes actually do look good + are good functionally.

I think style changes are annoying and I am happy for anyone to commit and push any changes directly, it might be easier!

File: src/widgets/Rectangle.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import { DANGER_RED } from '../globalStyles';
 
 const SIZES = {
-  large: { width: 140, height: 40 },
+  large: { width: 120, height: 40 },
```
Comment (mark-prins) [inline src/widgets/Rectangle.js:8]: I'm getting this on the emulator:

Screen Shot 2021-02-17 at 9 07 21 AM

maybe the font size needs to come down a bit too?

**Review by mark-prins — APPROVED** [review]
What have you done!?!!

that looks amazing! spacing works fine for me and the x-axis ticks really lift the look. perfect!

File: src/widgets/Rectangle.js (hunk 1 above)
Comment (mark-prins) [reply src/widgets/Rectangle.js:8]: taking it down to `20` is fine

**Comment by katherine-sussol** [comment]
🥳  Really like the x-axis updates, I think much clearer to read now. The whole chart looked pretty good on my emulator, the Rectangle text looks ok on mine (didn't notice wrapping) - but likely I am emulating a much larger device (Nexus 10), better to account for the smallest allowed 👍 

Screen Shot 2021-02-17 at 12 05 04 PM

**Comment by joshxg** [comment]
Awesome thanks ! 
597b404

**Pull request merged**